File: src/main/java/ru/dargen/evoplus/event/inventory/InventoryFillEvent.java
package ru.dargen.evoplus.event.inventory;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import net.minecraft.item.ItemStack;
import net.minecraft.screen.ScreenHandler;
import ru.dargen.evoplus.event.CancellableEvent;

import java.util.List;

@Getter @Setter
@AllArgsConstructor
public class InventoryFillEvent extends CancellableEvent {

    protected int syncId;
    protected List<ItemStack> contents;
    protected InventoryOpenEvent openEvent;
    protected ScreenHandler screenHandler;
    protected boolean hidden;

}

File: src/main/kotlin/ru/dargen/evoplus/api/event/inventory/InventoryFillEvent.kt
package ru.dargen.evoplus.api.event.inventory

import net.minecraft.item.ItemStack
import net.minecraft.screen.ScreenHandler

class InventoryFillEvent(
    syncId: Int,
    var contents: List<ItemStack>,
    var openEvent: InventoryOpenEvent?,
    var screenHandler: ScreenHandler,
    var isHidden: Boolean = false
) : InventoryEvent(syncId)
